$NetBSD: patch-ae,v 1.1 2008/08/17 08:40:50 dholland Exp $

Fix type mismatch through va_arg() and resulting nasal demons.

--- cs/csparse.c.orig	2005-06-30 21:04:34.000000000 -0400
+++ cs/csparse.c	2008-08-17 04:36:18.000000000 -0400
@@ -3359,9 +3359,9 @@ static NEOERR * _builtin_str_slice (CSPA
   NEOERR *err;
   char *s = NULL;
   char *slice;
-  int b = 0;
-  int e = 0;
-  int len;
+  long int b = 0;
+  long int e = 0;
+  long int len;
 
   result->op_type = CS_TYPE_STRING;
   result->s = "";
